.travis.yml

```diff
@@ -39,15 +39,21 @@ env:
 
 before_install:
   - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - npm install -g markdown-link-check
 
 script:
 - set -o pipefail
 
+- if [ $CHECK_MARKDOWN == "YES" ]; then
+      echo -en 'travis_fold:start:script.1\\r';
```

Comment:
Travis does to some auto-folding, but from the testing I've done it never seemed to fold this.
Yes it only gets triggered on the first build, I'll remove it for the next build and can see if you're happy to keep it unfolded after that!